repl/debug_commands.go

```diff
@@ -0,0 +1,24 @@
+package repl
+
+import (
+	"fmt"
+
+	"github.com/spacemeshos/CLIWallet/log"
+	gosmtypes "github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+func (r *repl) printAllAccounts() {
+
+	accounts, err := r.client.DebugAllAccounts()
+	if err != nil {
+		log.Error("failed to get debug all accounts: %v", err)
+		return
+	}
+
+	for _, a := range accounts {
+		fmt.Println(printPrefix, "Address:", gosmtypes.BytesToAddress(a.AccountId.Address).String())
+		fmt.Println(printPrefix, "Balance:", a.StateCurrent.Balance.Value, coinUnitName)
```

Comment:
I think you might get a null pointer exception here in some cases, be careful. If an account's balance is zero I think `Balance` might be missing. Same for Counter.

Reply:
good catch. fixed.